app/lib/widgets/cards/productCard/shared/price.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flex_ui/utils/extensions.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+import 'package:intl/intl.dart';
+
+enum Variant {
+  standard,
+  savings, // Defaults to text decoration linethrough and color 'disabled'
+}
+
+class Price extends StatelessWidget {
```

Comment:
We can start documenting the widgets

```
/// A flexible price display widget that handles various e-commerce pricing scenarios.
///
/// This widget can display prices in different formats and styles, supporting:
/// - Custom price formatting
/// - Multiple display variants (standard, savings)
/// - Theme customization
/// - Accessibility features
///
/// Example usage:
/// ```dart
/// FlexPrice(
///   price: 99.99,
///   variant: PriceVariant.standard,
///   formatterCallback: (price) => NumberFormat.currency(locale: 'en_US').format(price),
/// )
/// 
```